### PR TITLE
graph accordian moved in options

### PR DESCRIPTION
**Pull request by hrutik7**
## In the chart widget, group "show axes" , "show grid lines" and "loading state" inside one accordion named as "options"

image

fixes: #3261

### CODE REVIEW

**Comment by hrutik7** [comment]
sure

File: frontend/src/Editor/WidgetManager/widgetConfig.js [hunk 1]
```diff
@@ -237,6 +237,8 @@ export const widgets = [
         title: { value: 'This title can be changed' },
         markerColor: { value: '#CDE1F8' },
         showAxes: { value: '{{true}}' },
+       
+
```
Comment (gondar00) [inline frontend/src/Editor/WidgetManager/widgetConfig.js:241]: nit: 
@hrutik7 can you please remove these spaces on line num 240 & 241,

**Comment by hrutik7** [comment]
Sure thing

**Comment by hrutik7** [comment]
hey @gondar00 ,
I am not getting why one of the test is failing , I am new to opensource will you please look into this once the things get more clear to me I'll do more contributions
Thanks :)

**Comment by gondar00** [comment]
Hi @hrutik7, could you once try running the command `npm run --prefix frontend lint`

**Comment by withshubh** [comment]
Hey @hrutik7 
Please make the required changes so that we can merge this fix?

**Comment by hrutik7** [comment]
Hey @withshubh 
issue has been resolved , but I am getting this kind of errors

image

**Comment by withshubh** [comment]
@gondar00 Need your help here.

**Comment by gondar00** [comment]
@hrutik7  can u try running `docker-compose exec server npm i`

**Comment by withshubh** [comment]
@hrutik7 - Did you try running the command suggested above?

**Comment by hrutik7** [comment]
@withshubh yes but still getting the same problem
image

**Review by arpitnath — CHANGES_REQUESTED** [review]
please revert back changes added to plugins lock file and `packages/woocommerce/lib/operations.json`

**Comment by shivangT** [comment]
Hello @hrutik7 ,
Any updates on the following?
Please revert the changes in the mentioned file. We've to close the issue associated with this. 

Thanks

**Review by gondar00 — APPROVED** [review]

**Pull request merged**